crnk-core/src/main/java/io/crnk/core/engine/internal/dispatcher/controller/ResourceDelete.java

```diff
@@ -31,12 +31,12 @@ public boolean isAcceptable(JsonPath jsonPath, String method) {
 	@Override
 	public Result<Response> handleAsync(JsonPath jsonPath, QueryAdapter queryAdapter,
 										RepositoryMethodParameterProvider parameterProvider, Document requestBody) {
-		String resourceName = jsonPath.getElementName();
+		String resourcePath = jsonPath.getElementName();
 		PathIds resourceIds = jsonPath.getIds();
-		RegistryEntry registryEntry = context.getResourceRegistry().getEntry(resourceName);
+		RegistryEntry registryEntry = context.getResourceRegistry().getEntryByPath(resourcePath);
```

Comment:
getRegistryEntry helper method from below that  also performs a null check could also be used here (maybe has to be moved up to a controller superclass)